### ISSUE TITLE
Public availability of the health report

### ISSUE THREAD

**Issue by wafels** [aringlis/vso_health#5]
The health report should be publicly available somewhere. That would give users some idea of the accessibility of data via the VSO on a daily basis. The full history of performance should also be stored somewhere - all data generated by the health reporting code should be downloadable from somewhere. The health report will be available at least from the upcoming newly redesigned SDAC webpages.

**Comment by wafels** [comment]
Health report now available at https://docs.virtualsolar.org/wiki/VSOHealthReport . Closing
